### DISCUSSION TITLE
Installation failure after installing different versions - no error messages

### DISCUSSION THREAD

**Discussion by maraxai** [prisma/prisma#4258] [Q&A] [answered]
In the process of testing prisma, I installed prisma1 and prisma2 globally one after the other (in between uninstalling the prior version). This worked fine. Then, with prisma1 globally installed, I installed prisma2 --save-dev in a different directory. I guess this caused some havoc on my machine. 

Now, no matter what version I install, I cannot get it running. The installation seems complete (no error messages). The package.json has the entry but when I run **>prisma -v**, I get the message: _'prisma' is not recognized as an internal or external command, operable program or batch file.'_ (The same for prisma2). Installing and uninstalling only seems to affect the  package.json.

After the trouble I found this section in the docs: **Warning: If you have several Prisma projects on your machine, a global installation can lead to version conflicts between these projects.**

What do I have to do solve this problem? I used npm and yarn, I cleared the cache, I did several restarts, nothing helped. I am on Win 10.  

Any help is greatly appreciated.

**Comment by ryands17** [comment, marked as answer]
@maraxai 👋 

Remove your global installs for both Prisma 1 and Prisma2 (also `prisma2` is not the package for Prisma 2).

To install Prisma 2 locally, initialize a project with npm or yarn and install the following:

```
npm i @prisma/cli -D
npm i @prisma/client
```

You can also find it in the [getting started](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch-sql-typescript-postgres) section of the docs.

Now to run Prisma commands, use `npx` as follows:

```
npx prisma init
npx prisma migrate save --experimental
```

If you need Prisma 1 in another project, install it locally as well. The CLI for Prisma 1 is `prisma1`.

```
npm i prisma1 -D
```

So now you wont get any conflicts. Let me know if this works.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
